packages/app/src/views/Setup.vue

```diff
@@ -45,25 +37,22 @@ export default defineComponent({
   },
 
   setup() {
-    const router = useRouter();
-    const backendUrl = ref('');
+    const boxId = ref('');
```

Comment:
No type (ref<string>(''); 

Doesn't that fire a linting error?

Reply:
It implicitly takes the type from the empty string.